app/models/casa_case.rb

```diff
@@ -81,6 +81,10 @@ class CasaCase < ApplicationRecord
   # Validation to check timestamp and submission status of a case
   validates_with CourtReportValidator, fields: [:court_report_status, :court_report_submitted_at]
 
+  def latest_court_report
```

Comment:
I would like us to be cool and rails-y and make `latest_court_report` a scope :)
https://stackoverflow.com/questions/34363707/rails-active-record-create-a-scope-to-get-the-most-recent-child
https://stackoverflow.com/questions/5247886/rails-query-through-association-limited-to-most-recent-record
https://stackoverflow.com/questions/29321895/ruby-on-rails-how-to-get-latest-records-from-has-many-relationship
https://guides.rubyonrails.org/active_record_querying.html#scopes